src/plugins/field_formats/common/converters/duration.ts

```diff
@@ -50,9 +50,8 @@ function formatInputHumanPrecise(
   const valueInDuration = moment.duration(val * ratio, kind);
 
   return formatDuration(
-    val,
+    valueInDuration.as('seconds'),
```

Comment:
The changes look great, thanks!

I think we should also account for negative values like `-123` in the samples:

"Precise" formatting:
<img width="463" alt="Screenshot 2024-04-26 at 14 38 52" src="https://github.com/elastic/kibana/assets/1415710/2beba169-f125-4b7b-8a88-d92fec04a073">


While "approximate" formatting is:
<img width="437" alt="Screenshot 2024-04-26 at 14 39 06" src="https://github.com/elastic/kibana/assets/1415710/3886b36e-10da-4b6b-8e7b-5e2611c47d47">

Reply:
done in https://github.com/elastic/kibana/pull/181391/commits/67677d2147f10c87f7f2f3fca50cf10aed518c63